app/js/home/list-service/tests/list.service.test.js

```diff
@@ -0,0 +1,33 @@
+import { assert } from 'chai';
+
+import ListService from '../list.service.js';
+
+let service;
+
+describe('ListService (Unit Test)', function() {
+
+    beforeEach(function() {
+        let initialList = [];
+        service = new ListService(initialList);
+    });
+
+    it('should contain empty list after initialization', function() {
+        assert.equal(service.list.length, 0);
+    });
+
+    it('should add line', function () {
+        service.addLine('New Line');
+        assert.equal(service.list.length, 1);
+        assert.equal(service.list[0], 'New Line');
```

Comment:
`expect` in `*.spec.js` is different than `chai`'s assertion library's `expect`.
For example in `*.spec.js` to compare objects:
- `expect(service.list).toEqual(['New List']);`

and `chai`'s `expect`:
- `expect(service.list, "Should equal ['New Line']").to.deep.equals(['New Line']);`

See http://stackoverflow.com/questions/21396524/what-is-the-difference-between-assert-expect-and-should-in-chai for details between `chai`'s `expect`, `assert` and `should`.

I think that `expect` from 2 different libraries may be confusing. Let me know what do you think :)